### PR TITLE
Downtime webform updates

### PR DESCRIPTION
**Pull request by matyasselmeci**
These address some of the concerns from our run-through.

### CODE REVIEW

File: src/templates/generate_downtime_form.html.j2 [hunk 1]
```diff
@@ -99,14 +102,32 @@
...
+            {%- endif %}
+          </li>
+          <li>
+            {% if github -%}
+              Specify a commit message in the "Commit changes" section.
+            {%- else -%}
+              Commit your changes to the repository.
+            {%- endif %}
+            The commit message should look something like<br>
+            <code>Add downtime for &lt;RESOURCE(S)&gt; due to &lt;REASON&gt;</code>
```
Comment (brianhlin) [inline src/templates/generate_downtime_form.html.j2]: This form can only generate a single downtime so this should read `RESOURCE`

**Comment by brianhlin** [comment]
@matyasselmeci are you going to address resource FQDNs in a separate PR?

**Comment by matyasselmeci** [comment]
Ah, I _knew_ I was missing something. Nah, I'll do it in this one.

File: src/webapp/forms.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ class GenerateDowntimeForm(FlaskForm):
     start_time = TimeField("&nbsp;", [InputRequired()])
     end_date = DateField("End Date/Time (UTC)", [InputRequired()])
     end_time = TimeField("&nbsp;", [InputRequired()])
-    services = SelectMultipleField("Services (select one or more)", [InputRequired()], choices=[])
+    services = SelectMultipleField("Known OSG Services (select one or more)", [InputRequired()], choices=[])
```
Comment (brianhlin) [inline src/webapp/forms.py]: Finally found a resource with multiple services, maybe we should change this to `(select one or more with ctrl-click)`?

Comment (matyasselmeci) [reply src/webapp/forms.py]: I would've put that but it's not ctrl-click on macs.

**Pull request merged**